projects/Mallard/src/theme/appearance.ts

```diff
@@ -90,6 +90,13 @@ export const appAppearances: { [key in AppAppearance]: AppAppearanceStyles } = {
         color: color.text,
         dimColor: color.dimText,
     },
+    tertiary: {
+        backgroundColor: color.dimBackground,
+        cardBackgroundColor: color.dimBackground,
+        borderColor: '#052962',
```

Comment:
@RichieAHB @jimhunty Do these exist in our current colours?

Reply:
`#052962` doesn't exist if that's what you mean, and for the time being, if it's not used elsewhere then I feel like it can stay here ... unless that's not what you mean?

Reply:
Okay 👍 Thanks.